mriqc/workflows/functional.py

```diff
@@ -55,18 +55,19 @@ def fmri_qc_workflow(dataset, settings, name='funcMRIQC'):
     # Workflow --------------------------------------------------------
 
     # 1. HMC: head motion correct
-    hmcwf = hmc_mcflirt()
-    if settings.get('hmc_afni', False):
+    if settings.get('hmc_fsl', False):
```

Comment:
I would prefer to create one ```hmc_afni``` by default, and overwrite ```hmcwf``` with the ```hmc_mcflirt``` in case ```settings.get('hmc_fsl', False)``` is true.

I would keep the ```--hmc_afni``` flag though (doing nothing), just for backwards compatibility.